package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "wizard-sdk",
```

Comment:
Let's put it under @trustwallet

Reply:
Noted!

Reply:
looks that `package.json` name is a single word, if u are referring to the npm/gitgub package then it will be `@trustwallet/wizard-sdk` after release